src/app/tests/integration/chip_im_initiator.cpp

```diff
@@ -44,8 +47,8 @@ namespace {
 constexpr size_t kMaxCommandMessageCount = 3;
 constexpr size_t kMaxReadMessageCount    = 0;
 
-// The CHIP Message interval time in milliseconds.
-constexpr int32_t gMessageInterval = 1000;
+// The CHIP Message interval time in seconds.
+constexpr int32_t gMessageInterval = 1;
```

Comment:
How about renaming this to gMessageIntervalSeconds  and then the comment will not be necessary anymore?

It would also mean that reading the code below I would not need to reference the comment to validate units.